mobile/android/android-device.ts

```diff
@@ -4,11 +4,11 @@ import util = require("util");
 import Future = require("fibers/future");
 import path = require("path");
 import temp = require("temp");
-import byline = require("byline");
 import helpers = require("../../helpers");
 import os = require("os");
 import options = require("./../../options");
 import Fiber = require("fibers");
+import LogcatHelper = require("./LogcatHelper");
```

Comment:
`It is better to register `logcatHelper` to `injector`.
You need to call `$injector.require("logcatHelper", LogcatHelper)` at the end of `LogcatHelper.ts` file
and also you need to register it in bootstarap -> `$injector.register("logcatHelper", "./common/mobile/android/LogcatHelper.ts")`